### ISSUE TITLE
Lock screen is taking too long to appear when unlocking the phone

### ISSUE THREAD

**Issue by msbrogli** [HathorNetwork/hathor-wallet-mobile#44]
When the iPhone is unlocked, the wallet shows its balance before the lock screen shows up. Sometimes the balance even updates before the lock screen shows up.

It is happening on iPhone 7 Plus with Hathor Wallet v0.8.3.

**Comment by msbrogli** [comment]
@pedroferreira1 Maybe we have to put the lock screen when the app goes to background. I think we can add a handler that is called by the phone when either the app goes to background or the phone is locked. In this case, we can put either the lock screen or a "blank screen", just to prevent the balance from showing up.

What do you think?

**Comment by pedroferreira1** [comment]
Makes sense. I still need to check the code but we are probably redirecting to the Locked screen when the app is active again. We should redirect when the app goes to background.

**Comment by obiyankenobi** [comment]
Not necessarily when it goes to background. We actually only lock it if it's been in background for a period (I think 30s). That was our decision so the user wouldn't need to input the pin all the time, when he quickly switches apps.
